playwright/utils/mediaData.ts

```diff
@@ -0,0 +1,2 @@
+export const IMAGE_UUID = "4387c9f0-c53c-012f-9924-58d385a7bc34";
+export const VIDEO_UUID = "8820d790-e50c-0130-3a92-3c075448cc4b";
```

Comment:
i think if these are just used in one place, we can just keep em in the `page.ts` module for now - that will result in less indirection for the person reading the code.